sc-platform-api/src/api/user/user.controller.ts

```diff
@@ -1,4 +1,31 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get, Param } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { ApiEndpoint } from 'src/documentation/decorators';
+import { UserService } from './user.service';
+import { UserMapper } from 'src/mappers/user.mapper';
+import { UserDocumentation } from 'src/documentation/user';
+import { UserByIdPipe } from './user.pipe';
 
+@ApiTags('User')
+@Controller({
+  path: '/users',
+})
 @Controller('user')
-export class UserController {}
+export class UserController {
+  constructor (
+    private userService: UserService,
+    private userMapper: UserMapper,
+  ) {}
+
+  @ApiEndpoint({
+    summary: 'Return user\'s data',
+    documentation: UserDocumentation.GET_USER_BY_ID,
```

Comment:
Name of the documentation object must be the same as method's name
```suggestion
    documentation: UserDocumentation.GET_USER,
```